doc/api/events.md

```diff
@@ -1661,12 +1661,23 @@ console.log(listenerCount(myEmitter, 'event'));
 added:
  - v13.6.0
  - v12.16.0
+changes:
+  - version:
+    - v20.0.0
+    pr-url: https://github.com/nodejs/node/pull/41276
+    description: The `close`, `highWatermark`, and `lowWatermark`
+                 options are supported now.
 -->
 
 * `emitter` {EventEmitter}
 * `eventName` {string|symbol} The name of the event being listened for
 * `options` {Object}
   * `signal` {AbortSignal} Can be used to cancel awaiting events.
+  * `close` - {string\[]} Event names that will end the iteration.
+  * `highWatermark` - {integer} **Default:** `Number.MAX_SAFE_INTEGER`
+    The low watermark. The emitter is resumed every time size is lower than it.
+  * `lowWatermark` - {integer} **Default:** `1`
```

Comment:
The rest of the stream API uses the following case: `WaterMark` instead of `Watermark`.
Should the option be renamed before being documented, for consistency?

Reply:
That would be a good idea overall, I am just not sure if that would be considered a breaking change or not (it would definitely break code that uses the existing keys) and wether it is worth it or not

Reply:
That would be a breaking change for users like me who dig into undocumented things :)

However, backward compatibility could be maintained by internally supporting both cases, but documenting only the one that's consistent with other methods?

```js
const highWaterMark = options.highWaterMark ?? options.highWatermark ?? NumberMAX_SAFE_INTEGER;
validateInteger(highWaterMark, 'options.highWaterMark', 1);
const lowWaterMark = options.lowWaterMark ?? options.lowWatermark ?? 1;
validateInteger(lowWaterMark, 'options.lowWaterMark', 1);
```

Reply:
Yeah, I ended up doing that, but I think we might still want to change the docs to be explicit, so users with node 20.0.0 won't have any unexpected behavior 🙂